src/screens/Home.js

```diff
@@ -46,10 +53,23 @@ export default ({
     navigation.navigate('Learn');
   };
 
-  const openCategoryLearntPhrases = () => {
-    setCurrentCategory(LEARNT_PRHASES_ID);
-    // fetch Phrases for category
+  const openCategoryLearntPhrases = item => {
+    setCurrentCategory(item.id);
+    // fetch Phrases for categor
     setPhrases(learntPhrases);
+    if (learntPhrases.length === 0) {
+    } else {
+      navigation.navigate('Learn');
+    }
+  };
+
+  const openSeenPhrases = async item => {
+    setCurrentCategory(item.id);
+    setPhrases(seenPhrases);
+    if (seenPhrases.length === 0) {
+    } else {
+      navigation.navigate('Learn');
+    }
     navigation.navigate('Learn');
```

Comment:
same here something is wrong as on any condition it will try to navigate to 'Lern'
Probaby it should be
```
   if (seenPhrases.length !== 0) {

      navigation.navigate('Learn');
    }

```